cpp/ps0001/problem 2.cpp

```diff
@@ -1,11 +1,17 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
-int number1,number2,number3;
-cout << "Please Enter 3 numbers \n" << endl;
-cin >> number1>>number2>>number3;
-int sum=(number1+number2+number3);
- cout <<" sum equal\n "<<sum<<" \n";
-return 0;
-}
\ No newline at end of file
+  // prompting user for 3 numbers to add 1/2
+  unsigned short number1, number2, number3;
+  cout << "Please Enter 3 numbers \n" << endl;
+  cin >> number1 >> number2 >> number3;
+
+  // outputting sum of the 3 numbers 3/3
+  unsigned short = (number1 + number2 + number3);
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly